src/Evolution/DgSubcell/Actions/TciAndSwitchToDg.hpp

```diff
@@ -285,7 +285,7 @@ struct TciAndSwitchToDg {
         not cell_is_troubled and
 
         (((is_substep_method and
-           tci_history.size() == subcell_options.min_clear_tci_before_dg() - 1)
+           tci_history.size() >= subcell_options.min_clear_tci_before_dg() - 1)
```

Comment:
This looks reasonable, although I don't fully understand what the code is doing.  Should the same change be made for the multistep case 5 lines down?